pubspec.yaml

```diff
@@ -4,13 +4,13 @@ version: 2.0.3
 homepage: https://github.com/zeshuaro/google_api_headers
 
 environment:
-  sdk: ">=2.12.0 <4.0.0"
+  sdk: '>=2.19.6 <4.0.0'
   flutter: ">=1.20.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  package_info_plus: ^4.0.0
+  package_info_plus: ^4.2.0
```

Comment:
If we do not need functionalities from 4.2.0, I would prefer keeping this as is instead of bumping the required version

Reply:
My read of the changelog is there are bug fixes between those versions. Keeping legacy versions of kotlin and gradle can cause big problems for Android developers. This gradle/kotlin issue is why I made this PR in the first place. Android SDK version 34 is making big changes and legacy dependencies like this block people like me from upgrading.

I had to update a number of dependency projects like this one, so I don't exactly remember the specifics for this project though. Sorry.